### DISCUSSION TITLE
需要大佬帮助，同样的配置，换个订阅地址生成的ISO文件就无法启动了

### DISCUSSION THREAD

**Discussion by wwushi** [kkkgo/PaoPaoGateWay#142] [问和答]
![Snipaste_2024-11-15_13-26-24](https://github.com/user-attachments/assets/3626f218-53fd-4c08-9583-81effa6ba991)
![Snipaste_2024-11-15_13-26-28](https://github.com/user-attachments/assets/88421dab-b29a-436c-a651-25e37b84b10f)

network.ini：
ip=192.168.88.3
mask=255.255.252.0
gw=192.168.88.1
dns1=192.168.88.2

ppgw.ini：
#paopao-gateway

mode=suburl
net_rce=yes
max_rec=5000

fake_cidr=11.0.0.0/8

dns_ip=192.168.88.2

dns_port=5304

clash_web_port="80"

clash_web_password="password"

openport=yes

udp_enable=yes

sleeptime=30

socks5_ip="192.168.88.1"

socks5_port="7890"

ovpnfile="custom.ovpn"
ovpn_username=""
ovpn_password=""

yamlfile="custom.yaml"

suburl="订阅链接"
subtime=1d

fast_node=yes
test_node_url="http://www.gstatic.com/generate_204"
ext_node="Traffic|Expire| GB|Days|Date"
cpudelay="3000"

dns_burn=yes
ex_dns="192.168.88.2:53"

**Comment by kkkgo** [comment]
就是单纯的订阅地址访问不了。
